src/cpu/mod.rs

```diff
@@ -305,6 +457,19 @@ impl Cpu {
         self.set_reg(regop, result);
     }
 
+    fn load_register(&mut self, dest: ArithmeticOperand, src: ArithmeticOperand) {
+        self.set_reg(dest, self.get_reg(src));
+    }
+
+    fn load_register_hl(&mut self, regop: ArithmeticOperand) {
```

Comment:
Again this naming is confusing.  At very least we need comments, but I'd prefer clearer naming.  Even knowing what these instructions do, its hard to decipher.

Reply:
Jeez, I sure am embarrassed. I should have thought better about this...